apps/tests/msg/rpmsg-ping.c

```diff
@@ -121,21 +131,12 @@ int app (struct rpmsg_device *rdev, void *priv)
 
 	if (!i_payload) {
 		LPERROR("memory allocation failed.\r\n");
+		rpmsg_destroy_ept(&lept);
```

Comment:
I looked into that and it will introduce more churn than I'd think should be part of this series. Would it be better to address that issue in a separate PR?